apps/docs/pages/learn/auth-deep-dive/auth-google-oauth.mdx

```diff
@@ -24,7 +24,7 @@ How to add Google OAuth Logins to your Supabase Application.
 
 ### Logging in with external OAuth providers
 
-Connecting social logins such as Google, GitHub, or Facebook couldn't be easier. In this guide we'll walk you through the process of connecting Google, but the process is basically the same for all of the providers which includes: azure, bitbucket, github, gitlab, facebook, and google.
+Connecting social logins such as Google, GitHub, or Facebook couldn't be easier. In this guide we'll walk you through the process of connecting Google, but the process is basically the same for all of the providers which includes: azure (Microsoft), bitbucket, github, gitlab, facebook, and google.
```

Comment:
Can consider leaving this one as `azure` for consistency with the  bitbucket, github, facebook  which reflects the provider name that is passed in when visiting the authorize URL (e.g. https://<your-ref>.supabase.co/auth/v1/authorize?provider=<provider>)
```suggestion
Connecting social logins such as Google, GitHub, or Facebook couldn't be easier. In this guide we'll walk you through the process of connecting Google, but the process is basically the same for all of the providers which includes: azure, bitbucket, github, gitlab, facebook, and google.
```